### PR TITLE
Fix/normals

### PR DESCRIPTION
**Pull request by yalania**
- Removed some unused code

- multiply lights by TBN

As it is difficult to tell what is going on I have made some improvements on the workflow. 

- Added a reload shader button, so we can see changes not needing to open the engine evertytime
- Added adding normasl and lightmaps automatically when importing fbxs (if the path is correct)

https://drive.google.com/file/d/1faZTo4oPmGdn_wsuqAmC96QPeXChZ6Ev/view?usp=sharing

### CODE REVIEW

File: Engine/EditorUI/Panel/PanelScene.cpp [hunk 1]
```diff
@@ -135,6 +136,10 @@ void PanelScene::RenderSceneBar()
 			App->debug->show_debug_metrics = !App->debug->show_debug_metrics;
 		}
 
+		if (ImGui::Selectable("Reload shaders", false, ImGuiSelectableFlags_None, ImVec2(100, 0)))
+		{
+			App->program->LoadPrograms(SHADERS_PATH);
+		}
```
Comment (raulgonzalezupc) [inline Engine/EditorUI/Panel/PanelScene.cpp:142]: Doesn't work :/

File: Game/Resources/Shaders/blinn_phong.fs [hunk 2]
```diff
@@ -277,11 +276,10 @@ vec3 CalculateNormalMapAndLiquid(const Material material, const vec2 tiling)
...
 }
 vec3 CalculateDirectionalLight(const vec3 normalized_normal, vec4 diffuse_color, vec4 specular_color, vec3 occlusion_color, vec3 emissive_color)
-{
-	
+{   
 	vec3 view_pos    = transpose(mat3(matrices.view)) * (-matrices.view[3].xyz);
-	vec3 view_dir    = normalize(view_pos - position);
-	vec3 light_dir   = normalize(-directional_light.direction );
+	vec3 view_dir    = 	TBN* normalize(view_pos - position);
+	vec3 light_dir   = 	TBN* normalize(-directional_light.direction );
```
Comment (raulgonzalezupc) [inline Game/Resources/Shaders/blinn_phong.fs:282]: why only on directional light?

File: Engine/EditorUI/Panel/PanelScene.cpp (hunk 1 above)
Comment (yalania) [reply Engine/EditorUI/Panel/PanelScene.cpp:142]: how did you try it?

Comment (raulgonzalezupc) [reply Engine/EditorUI/Panel/PanelScene.cpp:142]: i changed some code on the FS and then pressed the button

**Comment by yalania** [comment]
Close as we are using ligh branch now.

**Pull request closed without merging**